components/servo/webview_delegate.rs

```diff
@@ -418,6 +418,17 @@ pub trait WebViewDelegate {
     /// Request to stop a haptic effect on a connected gamepad.
     fn stop_gamepad_haptic_effect(&self, _webview: WebView, _: usize, _: IpcSender<bool>) {}
 
+    fn show_select_element_menu(
+        &self,
+        _webview: WebView,
+        _options: Vec<String>,
+        _selected_option: Option<usize>,
+        _position: DeviceIntRect,
+        ipc_sender: IpcSender<Option<usize>>,
+    ) {
+        let _ = ipc_sender.send(None);
+    }
+
```

Comment:
Please modify this to use a responder pattern objection instead of an `IPCSender`. In addition, I think it makes sense to expose a `MenuItem` struct here to hide the details of what exactly a menu item is -- in case we want to, at some point, hide an id somewhere inside. Perhaps the same struct can be used for context menus in the future.

Reply:
Sure, I'll create something similar to https://github.com/servo/servo/blob/main/components/servo/webview_delegate.rs#L22-L49 for select element prompts.